apps/antalmanac/src/components/dialogs/About.tsx

```diff
@@ -0,0 +1,58 @@
+import { useCallback } from 'react';
+import {
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogContentText,
+    DialogTitle,
+    type DialogProps,
+} from '@mui/material';
+import { MuiMarkdown } from '$components/MuiMarkdown';
+
+const markdown = `
+AntAlmanac is a schedule planning tool for UCI students.
+
+
+Interested in helping out? Join our [Discord](https://discord.gg/GzF76D7UhY)
+or checkout the [code on GitHub](https://github.com/icssc/AntAlmanac).
+
+
+This website is maintained by the [ICS Student Council](https://studentcouncil.ics.uci.edu/)
+Projects Committee and built by students from the UCI community.
+
+<a target="_blank" href="https://github.com/icssc/AntAlmanac/contributors">
+    <img
+        src="https://contrib.rocks/image?repo=icssc/antalmanac"
+        width='100%'
+        alt="AntAlmanac Contributors"
+    />
+</a>
+`;
```

Comment:
A neat DX improvement because of this is that we can store our patch notes as markdown files somewhere, and have a process of importing them as raw text into the app. Then we can just render each patch notes with the markdown component